src/main/java/nextstep/subway/ui/LineController.java

```diff
@@ -48,16 +53,32 @@ public ResponseEntity<LineResponse> readLine(@PathVariable long id) throws NotFo
     }
 
     @DeleteMapping("/{id}")
-    public ResponseEntity<LineResponse> deleteLine(@PathVariable long id) {
+    public ResponseEntity deleteLine(@PathVariable long id) {
         lineService.deleteLine(id);
         return ResponseEntity.noContent().build();
     }
 
     @PutMapping("/{id}")
-    public ResponseEntity<LineResponse> updateLine(@PathVariable long id, @RequestBody LineRequest lineRequest)
+    public ResponseEntity updateLine(@PathVariable long id, @RequestBody LineRequest lineRequest)
             throws NotFoundException {
-        LineResponse lineResponse = lineService.updateLine(id, lineRequest);
+        lineService.updateLine(id, lineRequest);
+        return ResponseEntity.ok().build();
+    }
+
+    @PostMapping("/{lineId}/sections")
+    public ResponseEntity<LineResponse> addSection(@PathVariable Long lineId,
+                                                   @RequestBody SectionRequest sectionRequest) {
+        LineResponse lineResponse = sectionService.addSection(sectionRequest, lineId);
```

Comment:
```suggestion
    @PostMapping("/{lineId}/sections")
    public ResponseEntity addSection(@PathVariable Long lineId,
                                                   @RequestBody SectionRequest sectionRequest) {
        LineResponse lineResponse = sectionService.addSection(sectionRequest, lineId);
```
![스크린샷 2022-06-16 오전 1 27 19](https://user-images.githubusercontent.com/31276206/173878172-1745c33d-6692-4526-8b7d-c14fd4d3e25b.png)

Reply:
요구사항과 달랐군요... 반영하였습니다.